File: api/src/structs/account/account.go
package structs

import "time"

type Account struct {
	ID                     string
	Name                   string
	Email                  string
	Street                 string
	City                   string
	Zip                    string
	CountryCode            string
	CompanyNumber          string
	VatNumber              string
	PaidTo                 string
	PlanRefer              string
	Role                   string
	Parent                 string
	Password               string
	Newsletter             bool
	NewsletterConfirmation time.Time
}

File: frontend/model/Account.ts
export default interface Account {
  Name: string;
  Parent: string;
  Email: string;
  Role: string;
  CountryCode: string;
  Id: string;
  Street?: string;
  City?: string;
  Zip?: string;
  CompanyNumber?: string;
  VatNumber?: string;
  Newsletter?: boolean;
  PaidTo?: string;
}
